### ISSUE TITLE
Prevent display of Secret Key during onboarding when going back

### ISSUE THREAD

**Issue by Eshwari007** [blockstack/blockstack-app#2278]
The user  is testing version v3.2.0

- An user signs in with a secret key
- Upon receiving the password screen, the user realized that it was a wrong account
- User selects the back arrow and it landed on "Back up your secret key page" instead of the "Sign in with your secret key page"

Attaching screen recording for perusal

https://user-images.githubusercontent.com/38872890/156217576-e4c9e116-b4e4-4527-9630-2abbd594618c.mov

The issue was found by @andresgalante hence reporting it accordingly.

**Comment by markmhendrickson** [comment]
Good catch. This is unexpected behavior and a possible security issue, because the key is shown in plain text on the previous screen, which might be exposed inadvertently upon going back.
